src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Linq;
```

Comment:
fwiw: linq concerns me in classificaiton. it's a hotspot, so we try to avoid unnecessary allocatinos.